submissions/ses3332002/tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,97 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Being {
+  constructor (name, friends, species) {
+    this.species = species;
+    this.name = name;
+    this.friends = friends;
+  }
+  
+  getDescription() {
+    return Object.entries(this)
+    .filter(([key]) => propsToPrint.includes(key))
+    .map(([,value]) => Array.isArray(value) ? value.join(', ') : value)
+    .join('; ');
```

Comment:
`Object` methods like `keys`, `values`, `entries` as well as `for...in` shouldn't be used as these do not guarantee any particular order of keys/values

Please specify the list of properties explicitly.

Reply:
OK, I will remake it.

Reply:
I remade it.